Modified release instructions for MacOS users (#5034)
operator-framework/operator-sdk: website/content/en/docs/contribution-guidelines/releasing.md

brew install gnu-sed make
```

Verify the version of `make` using command `make --version`.

> Here, `brew` automatically installs `3.81` version of `make`. In order to update it to `4.3` use below command.

Reviewer: ```suggestion
For MAC users, `brew` automatically installs `3.81` version of `make`. In order to update it to `4.3` use below command.
```

Reviewer: ```suggestion
Here, `brew` automatically installs `3.81` version of `make`. Make sure that it is upgraded to `4.3+`. 
```

Author: modified.

> 

Reviewer: ```suggestion
To use `gnu make` as default, add it to the `PATH`:
```

Author: Added above line.

```sh

> PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"

Reviewer: this is for adding `gnubin` to PATH, right ?

Author: yes. this will upgrade make version.

> echo 'export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"' >> ~/.bashrc

Reviewer 2: ```suggestion
echo 'export PATH="/usr/local/opt/make/libexec/gnubin:$PATH"' >> ~/.bashrc
```

Author: Oh! placed at wrong place. Modified.

```

Verify that the version of `sed` is higher than 4.3 using command `gnu-sed --version`.

Add the `gnubin` directory to your PATH from your `~/.bashrc`:

```sh

> echo 'export PATH="/usr/local/opt/make/libexec/gnubin:$PATH"' >> ~/.bashrc

Author: Modified.

brew install gnu-sed make
```

> Verify the version of `make` using command `make --version`.
> 
> For MAC users, `brew` automatically installs `3.81` version of `make`. Make sure that it is upgraded to `4.3+`.
> To use `gnu make` as default, add it to your `PATH`:

Reviewer 2: ```suggestion
Verify that the version of `make` is higher than 4.2 using command `make --version`.

Add the gnubin directory to your PATH from your `~/.bashrc`:
```

Author: Modified.

```sh

> PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"

Reviewer 2: ```suggestion
echo 'export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"' >> ~/.bashrc
```

Author: Modified.

```

> Also, verify the version of `gnu-sed` using command `brew info gnu-sed`. `Brew` installs GNU `sed` as `gsed`.
> 
> If you need to use it as `sed`, you can add a `gnubin` directory to your PATH from your bashrc like:

Reviewer 2: You always need to use it as sed
```suggestion
Verify that the version of `sed` is higher than 4.3 using command `gnu-sed --version`.

Add the `gnubin` directory to your PATH from your `~/.bashrc`:
```

Author: Modified according to suggestion.

```sh

> export PATH="/usr/local/opt/make/libexec/gnubin:$PATH"

Reviewer 2: ```suggestion
echo 'export PATH="/usr/local/opt/make/libexec/gnubin:$PATH"' >> ~/.bashrc
```

Author: Modified.
